builtin/providers/aws/resource_aws_emr.go

```diff
@@ -0,0 +1,413 @@
+package aws
+
+import (
+	"log"
+
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform/helper/schema"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func resourceAwsEMR() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEMRCreate,
+		Read:   resourceAwsEMRRead,
+		Update: resourceAwsEMRUpdate,
+		Delete: resourceAwsEMRDelete,
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"release_label": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"master_instance_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"core_instance_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
By default, "core_instance_type" will be the same as "master_instance_type", 
- [x] As you suggested make it into "Computed"